File: Java-Core/src/main/java/presto/python/PythonOrdinalArgument.java
package presto.python;

import presto.utils.CodeWriter;

public class PythonOrdinalArgument implements PythonArgument {

	PythonExpression expression;
	
	public PythonOrdinalArgument(PythonExpression expression) {
		this.expression = expression;
	}

	@Override
	public void toDialect(CodeWriter writer) {
		expression.toDialect(writer);
	}

}

File: Java-CodeServer/src/main/resources/js/prompto/python/PythonOrdinalArgument.js
function PythonOrdinalArgument(expression) {
    this.expression = expression;
    return this;
}

PythonOrdinalArgument.prototype.toDialect = function(writer) {
    this.expression.toDialect(writer);
};

exports.PythonOrdinalArgument = PythonOrdinalArgument;
